firmware/shared/src/app/App_SharedMacros.h

```diff
@@ -64,3 +64,5 @@ typedef struct \
 {\
     unsigned char _unused;\
 } name;
+
+#define FLOOR_TO_MULTIPLE(x, multiple) (x / multiple) * multiple
```

Comment:
nit: macro function should be wrapped in brackets ()

Reply:
whoops looked like i never ended up using, deleted